app/lib/SecvisogramPage.js

```diff
@@ -385,7 +385,7 @@ function createFileName(doc, isValid, extension) {
   if (trackingId.trim().length === 0) {
     trackingId = 'csaf_2_0'
   } else {
-    trackingId = trackingId.replace(/([^a-z0-9]+)/gi, '_')
+    trackingId = trackingId.replace(/([^a-z0-9+\-_]+)/gi, '_')
```

Comment:
Nit: Why would we want to replace an underscore with itself?

Reply:
Actually, we don't. But as the regex starts with `^` after the `[` the `_` should not match...

Reply:
Silly me, yes, obviously 😅

Reply:
No worries!